client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -173,7 +174,7 @@ class JetpackSiteStats extends Component {
 					</FormFieldset>
 				</FoldableCard>
 
-				<CompactCard href={ '/stats/day/' + siteId }>
+				<CompactCard href={ getStatsPathForTab( 'day', siteId ) }>
```

Comment:
Should the slug be preferred here? As a user, I prefer to see the slug rather than some numbers. `getStatsPathForTab` seems to accept either: 

```js
function getStatsPathForTab( tab, siteIdOrSlug ) {
```